recipes/folly/all/conanfile.py

```diff
@@ -22,10 +22,12 @@ class FollyConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "with_sse4_2" : [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
+        "with_sse4_2" : True
```

Comment:
```suggestion
        "use_sse4_2" : True
```

with is not a good prefix, as it's not a dependency. Prefer `use` instead: https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md#recommended-feature-options-names

Reply:
thanks. fixed.